analyses/cms-open-data-ttbar/README.md

```diff
@@ -0,0 +1,23 @@
+#### Instructions for paired notebook
+
+If you only care about running the `ttbar_analysis_pipeline.ipynb` notebook, you can completely ignore the `ttbar_analysis_pipeline.py` file.
+
+This notebook (`ttbar_analysis_pipeline.ipynb`) is paired to the file `ttbar_analysis_pipeline.py` via Jupytext (https://jupytext.readthedocs.io/en/latest/). Using `git diff` with this file instead of the `.ipynb` file is much simpler, as you don't have to deal with notebook metadata or output images. However, in order for the notebook output to be preserved, the notebook still needs to be version controlled. It is ideal to run `git diff` with the option `-- . ':(exclude)*.ipynb'`, so that `.ipynb` files are ignored. 
+
+The `.py` file can also be run as a Python script.
```

Comment:
Here we (currently) will still have the problem with the `async` code, right?

Reply:
What problem? I am not sure what you are referring to

Reply:
This is ServiceX-specific, the `async def produce_all_the_histograms()` section is needed to run successfully as a script (so the notebook version needs to be commented out and that other piece needs to be enabled).

This issue will go away via #96, so I realize now that it won't matter here.